matteodelabre/toltec
# [display] add remux.service.conf override to display package on rm2

this turns on the display shim for remux on rm2 apps. the reasoning for this is so that people can launch homebrew apps that they compiled from other sources without having to use patchelf. 


TEST PLAN:

* install display package on rm2, verify that /etc/systemd/system/remux.service.d/remux.service.conf exists and has an Environment= line in it
* restart remux, verify that applications can be launched
* custom compile an application without patchself (or remove the DT needed using patchelf) and verify remux can launch it
